app/helpers/search_helper.rb

```diff
@@ -88,6 +88,15 @@ def raro_input_as(name,type,opts)
         raro_text_field(name, opts)
       when :range
         raro_range(name)
+      when :monthyear
+        raro_monthyear(name, opts)
+      when :check_boxes
+        if opts.key?(:collection)
+          collection = opts[:collection]
+        else
+          collection = opts[:collection_if].call
+        end
```

Comment:
Pra ficar consistente com o resto do código, essa parte de pegar a `collection` deveria ser similar ao que é feito no `select`:
```ruby
if opts[:collection_if] and opts[:collection_if].class == Proc
  opts[:collection] = ActionView::Helpers::FormBuilder.instance_eval(&opts[:collection_if])
end
```

E pra evitar duplicação de código, poderia colocar isso no início do método, e tirar de dentro do `case` do select.